package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ubiquibot/configuration",
-  "version": "1.1.0",
+  "version": "2.0.1",
```

Comment:
Why not 2.0.0 ?

Reply:
To make sure the package works properly I published `2.0.0` so to avoid conflicts with `release-please` I bumped it one patch version up.

Reply:
And afterwards had to fix a small bug, so here we are in `2.0.2`
https://www.npmjs.com/package/@ubiquibot/configuration